docs/content/docs/build/auth.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Authentication & authorization"
+menuPosition: 3
+editor: tylerbutler
+---
+
+Security is critical to modern web applications. Fluid Framework, as a part of your web application architecture, is an
+important piece of infrastructure to secure. Fluid Framework is a layered architecture, and auth-related concepts are
+primarily a concern of the *driver* layer (see [Architecture]({{< relref "architecture.md" >}})). This means that the
+specifics of authentication could differ based on the Fluid service and its driver.
+
+The information below is based on Azure Fluid Relay service but also applies to
```

Comment:
The acronym is likely going to be knowledge that the developer has by the time they reach this doc but I don't think we mention it specifically in this one before we refer to it so it might be helpful to just add that in here as well
```suggestion
The information below is based on Azure Fluid Relay service (FRS) but also applies to
```